R/pipe.R

```diff
@@ -188,15 +188,15 @@ pipe <- function()
 #' @export
 `%>%`  <- pipe()
 
-#' magrittr compound assignment pipe-operator
+#' compound assignment pipe
```

Comment:
Should start sentence with capital letter

Reply:
Will capitalize them all (did the same for all 4 pipes). I was trying to respect the original, but I wasn't sure if the lower case had been deliberate or a way not to change the package name.

Reply:
magrittr is a proper name that is not capitalised, so it doesn't get capitalised even when at the start of a sentence.

Reply:
Right

Reply:
Corrected